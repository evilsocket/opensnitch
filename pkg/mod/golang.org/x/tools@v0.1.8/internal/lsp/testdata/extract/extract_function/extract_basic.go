package extract

func _() { //@mark(exSt25, "{")
	a := 1    //@mark(exSt1, "a")
	_ = 3 + 4 //@mark(exEn1, "4")
	//@extractfunc(exSt1, exEn1)
	//@extractfunc(exSt25, exEn25)
} //@mark(exEn25, "}")
