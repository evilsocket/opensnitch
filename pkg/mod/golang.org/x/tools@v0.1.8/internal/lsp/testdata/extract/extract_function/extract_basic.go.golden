-- functionextraction_extract_basic_3_10 --
package extract

func _() { //@mark(exSt25, "{")
	//@mark(exSt1, "a")
	newFunction() //@mark(exEn1, "4")
	//@extractfunc(exSt1, exEn1)
	//@extractfunc(exSt25, exEn25)
}

func newFunction() {
	a := 1
	_ = 3 + 4
} //@mark(exEn25, "}")

-- functionextraction_extract_basic_4_2 --
package extract

func _() { //@mark(exSt25, "{")
	//@mark(exSt1, "a")
	newFunction() //@mark(exEn1, "4")
	//@extractfunc(exSt1, exEn1)
	//@extractfunc(exSt25, exEn25)
}

func newFunction() {
	a := 1
	_ = 3 + 4
} //@mark(exEn25, "}")

